source/about/accessibile_html_error_messages.md

```diff
@@ -1,58 +1,58 @@
+### What is TeXML? 
+TeXML is an open source process that describes the markup elements of TeX using XML syntax. This allows XML data to be presented in PDF.
 
-TeX and LaTex are open source software programs and are supported, maintained and iterated upon by the communities that use them. As a result, there are many packages available offering a myriad of features to format research papers. 
+### What is LaTeXML?
+LaTeXML is an open source software package that converts LaTeX documents to XML, HTML, EPUB, JATS and TEI. 
 
-TeX and LaTex packages are platform independent which allows for the ability to view documents regardless of the operating system or software set up making collaboration simple.
+### Why are TeX and LaTeX so popular? 
+They are beloved by academic communities for their ability to easily display complex mathematical formulae, symbols, charts and graphs that common word processing programs have difficulty with. 
+
+TeX and LaTeX are open source software programs  that  are platform independent which allows for the ability to view documents regardless of the operating system or software set-up making collaboration simple. These programs are maintained and iterated upon by the communities that use them, and as a result, there are many packages available offering a myriad of features to format research papers. 
 
-### Why are TeX and LaTex so difficult to support?
-Due to the robust open source community supporting and developing TeX and LaTex packages, there are an enormous number of packages in use. We are continuously reviewing which features and packages are most popular in an effort to provide error free, accessible HTML papers.
+### Why are TeX and LaTeX so difficult to support?
+Due to the robust open source community supporting and developing TeX and LaTeX packages, there are many packages in use. NIST is continuously reviewing which features and packages are most popular to add to the LaTeXML converter in an effort to provide error free, accessible HTML papers.
 
-### Which TeX/LaTeX packages do you or do not support?
+### Which TeX/LaTeX packages are supported?
 - Please see our list of supported packages
 - Please see our list of [unsupported packages](https://github.com/brucemiller/LaTeXML/wiki/Porting-LaTeX-packages-for-LaTeXML)
 
-## Do you support TikZ?
-See What are 'partially' supported packages?
+### Is TikZ supported?
+Sort of. See What are 'partially' supported packages?
+
+### What are ‘partially’ supported packages? 
 
-## What are ‘partially’ supported packages? 
+A partially supported package is a package that has select features NIST has chosen to support in LaTeXML. For example, TikZ offers the ability to … (need to add list of features) and LaTeXML supports …. (add list of features supported) However, TikZ also provides the ability to… which LaTeXML does not support.
 
-A partially supported package is a package that has select features we support in research papers.
+### What is NIST?
```

Comment:
Do we need this section? There is a link to NIST in an earlier section. That seems like enough.

Reply:
One more recommendation: a section for authors about how they can help reduce HTML errors and make papers more accessible by using packages that LaTeXML fully supports.